misk/src/main/kotlin/misk/eventrouter/RealEventRouterModule.kt

```diff
@@ -30,7 +30,8 @@ class RealEventRouterModule(val environment: Environment) : KAbstractModule() {
     }
     bind<ClusterMapper>().to<ConsistentHashing>()
     install(MoshiAdapterModule(SocketEventJsonAdapter))
-    multibind<WebActionEntry>().toInstance(WebActionEntry(EventRouterConnectionAction::class))
+    multibind<WebActionEntry>().toInstance(
```

Comment:
nit, but the line length here seems a little aggressively short.

Reply:
Reformatted so they fit on single line